README.md

```diff
@@ -1 +1,162 @@
-# Hello-Microverse-Project
\ No newline at end of file
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+  
+  <h3><b>Hello Microverse Project </b></h3>
+
+</div>
```

Comment:
Please, Kindly remove this `div` and its content as it wasn't included in the Microverse Readme Template you were asked to use 😸  
[https://github.com/microverseinc/readme-template](url)